jena-tdb/src/main/java/org/apache/jena/tdb/base/block/BlockMgrCache.java

```diff
@@ -58,10 +58,7 @@ private BlockMgrCache(int readSlots, int writeSlots, final BlockMgr blockMgr)
     {
```

Comment:
If this whole class is only for 32-bit operation, then I reckon maybe I should put `Cache0` back in and move on. It doesn't seem like a good place to invest time, no?